templates/docker-compose/.env

```diff
@@ -33,7 +33,6 @@ HOST=example.com
 # Misc
 APP_VERSION=3.6.2
 MAILTO=example@supportpal.com
-DOMAIN_NAME=example.com
```

Comment:
Not sure about renaming `.env.dist` to `.env`. Some stuff like `MAILTO` does need to be changed by the user?